lib/indexer/comparer.rb

```diff
@@ -21,8 +21,8 @@ def run
       log "processing: #{@old_index_name}/#{@new_index_name}"
 
       search_body = {}
-      if @filtered_format
-        search_body[:post_filter] = { term: { format: @filtered_format } } if @filtered_format
+      if @filtered_format && @filtered_format
```

Comment:
This is checking `@filtered_format` twice, so I think the second clause can be removed.